psst-gui/Cargo.toml

```diff
@@ -11,26 +11,27 @@ repository = "https://github.com/jpochyla/psst"
 psst-core = { path = "../psst-core" }
 
 crossbeam-channel = "0.5"
-druid-shell = { git = "https://github.com/jpochyla/druid", branch = "psst", features = ["raw-win-handle"] }
 druid = { git = "https://github.com/jpochyla/druid", branch = "psst", features = ["im", "image", "jpeg", "png", "serde"] }
+druid-enums = { git = "https://github.com/luleyleo/druid-enums" }
+druid-shell = { git = "https://github.com/jpochyla/druid", branch = "psst", features = ["raw-win-handle"] }
 env_logger = "0.9"
 fs_extra = "1.2"
 itertools = "0.10"
 log = "0.4"
 lru-cache = "0.1"
-once_cell = "1.8"
+once_cell = "1.9"
 open = "2.0"
 parking_lot = "0.11"
 platform-dirs = "0.3"
 rand = "0.8"
-raw-window-handle = "0.3"
+raw-window-handle = "0.4"
```

Comment:
this version bump needs to be reflected on the druid fork too. having issues building on Windows.

```
error[E0599]: the method `raw_window_handle` exists for reference `&WindowHandle`, but its trait bounds were not satisfied
   --> psst-gui\src\controller\playback.rs:132:43
    |
132 |                 let handle = match window.raw_window_handle() {
    |                                           ^^^^^^^^^^^^^^^^^ method cannot be called on `&WindowHandle` due to unsatisfied trait bounds
    |
   ::: C:\Users\gordo\.cargo\git\checkouts\druid-64a54200c8cd3ebf\625374b\druid-shell\src\window.rs:173:1
    |
173 | pub struct WindowHandle(pub(crate) backend::WindowHandle);
    | ---------------------------------------------------------- doesn't satisfy `WindowHandle: HasRawWindowHandle`
    |
    = note: the following trait bounds were not satisfied:
            `WindowHandle: HasRawWindowHandle`
            which is required by `&WindowHandle: HasRawWindowHandle`
    = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
    |
1   | use raw_window_handle::HasRawWindowHandle;
    |
```

Reply:
Ooops, let's revert it then.